src/ppom-controller.test.ts

```diff
@@ -390,25 +394,23 @@ describe('PPOMController', () => {
       });
     });
 
+    // TODO: This test description is wrong. Rename to something that more matches the test being run
     it('should not fail even if local storage files are corrupted and CDN also not return file', async () => {
       buildFetchSpy();
-      ppomController = buildPPOMController({
-        storageBackend: buildStorageBackend({
-          read: async (): Promise<any> => {
-            throw new Error('not found');
-          },
-        }),
-      });
+      ppomController = buildPPOMController();
+
       jest.runOnlyPendingTimers();
 
       await ppomController.usePPOM(async (ppom: any) => {
         expect(ppom).toBeDefined();
         return Promise.resolve();
       });
       jest.runOnlyPendingTimers();
-      buildFetchSpy(undefined, {
-        status: 500,
+
+      ppomController.update((draftState: PPOMState) => {
+        draftState.chainStatus = {};
       });
```

Comment:
`update` should be private method

Reply:
is there a way to ensure chainStatus is empty? Because this code in constructor of controller will mean there will always be at least one chain status

```
      chainStatus: state?.chainStatus ?? {
        [currentChainId]: {
          chainId: currentChainId,
          lastVisited: new Date().getTime(),
          dataFetched: false,
          versionInfo: [],
        },
      },
```